helpers/labgrid-raw-interface

```diff
@@ -85,9 +96,10 @@ if __name__ == "__main__":
     parser.add_argument('program', type=str, help='program to run, either tcpreplay or tcpdump')
     parser.add_argument('interface', type=str, help='interface name')
     parser.add_argument('count', nargs="?", type=int, default=None, help='amount of frames to capture while recording')
+    parser.add_argument('--duration', type=int, default=None, help="Amount of time to capture while recording")
```

Comment:
Is there a reason for calling this "duration" instead of "timeout"?

Reply:
duration was chosen because it's distinct from the `timeout` argument used in `rawnetworkinterface.py`. It can be confusing have the same name meaning different things in that file, so I thought "duration" was a more descriptive name